ls1intum/Artemis
# `Programming exercises`: Fix an issue with Java Maven sequential test runs

### Checklist
#### General



#### Changes affecting Programming Exercises

### Motivation and Context



Maven sequential test runs always fail. 

### Description


The pom.xml within the `structural` and `behavioral` directories of the test repositories are missing some plugins, thus they always fail.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Student

1. Log in to Artemis as Instructor
2. Create Java Maven Exercise with Sequential Test Runs enabled
3. Build results should be as expected (0% template, 100% solution)
4. As student, submit to Programming Exercise
5. Results should be as expected

### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests
